### ISSUE TITLE
Fortnite

### ISSUE THREAD

**Issue by Phelyxx** [fqlx/XboxKeyboardMouse#125]
Hi, when I use XboxKeyboardMouse on Fortnite I feel my mouse sensitivity so weird. Its like robot movements and too dificult to use. I´m looking for a profile of fortnite and how do u build with this. I speak spanish, sorry for my bad english xD

**Comment by DavidRieman** [comment]
Did you try the calibration process?  Also, you can change the sensitivity in the settings; make sure to save the settings after changing the value.  See more details in the readme.

**Comment by Phelyxx** [comment]
I tried the calibration process and different configurations, but it´s so weird anyways. Can you please try to config it in fortnite and send me your config?

**Comment by xboxkeyboardmouse** [comment]
Yep it does not work well on fortnite it feels choppy and inconsistent no matter what you do

**Comment by GustauvOfUganda** [comment]
Is it a wireless mouse

**Comment by DavidRieman** [comment]
Yeah a developer here will probably need to try out XKM with Fortnite to figure out if there's an issue here, given there are a number of reports specifically against it, and given its popularity.  (I'll close some of the others to cite this issue.)
